targets/PythonSdk/make.js

```diff
@@ -30,6 +30,10 @@ exports.makeCombinedAPI = function (apis, sourceDir, apiOutputDir) {
     console.log("Generating Combined Client/Server api from: " + sourceDir + " to: " + apiOutputDir);
 
     templatizeTree(locals, path.resolve(sourceDir, "source"), apiOutputDir);
+
+    var iniTemplate = getCompiledTemplate(path.resolve(sourceDir, "templates/__init__.py.ejs"));
```

Comment:
This is not necessary if you leave it in source/playfab and let templatizeTree do the work.